Binary Search Adding

prakharsharma27 opened the pull request:
Binary Search addition in this Repository

grzesiekkedzior reviewed (changes requested):
The name of class is wrong, no license, indentation, tests... Please read carefully the README file and correct the rest. Take a look at the other classes in this project.

prakharsharma27 commented:
I have added License , Corrected class name and other than that i don't understand the issue you raised. 
Please elaborate what test and indentation you are asking for .
As i have already added file to test this BinarySearchSnippet named BinarySearchTest.java. And code is well indented 
Kindly explain what to correct further.

grzesiekkedzior commented:
This project is written in [google style.](https://google.github.io/styleguide/javaguide.html).Tests... I think that this example below show you everything. You didn't write a test but interface which simulate test. The [tests](https://junit.org/junit5/docs/current/user-guide/) are invented to avoid that situation. Therefor we have [assertion.](https://junit.org/junit5/docs/5.6.2/api/org.junit.jupiter.api/org/junit/jupiter/api/Assertions.html)
```java
  Assert.assertEquals(expected,actual);
```
If you build a project, you should get an checkstyle error that show you what is wrong.

prakharsharma27 commented:
please do tell me if its ok now
